packages/block-editor/src/components/image-size-control/index.js

```diff
@@ -137,7 +137,11 @@ export default function ImageSizeControl( {
 								<ToggleGroupControlOption
 									key={ scale }
 									value={ scale }
-									label={ `${ scale }%` }
+									label={ sprintf(
+										/* translators: Percentage value. */
+										__( '%1$s%%' ),
```

Comment:
```suggestion
										__( '%1$d%%' ),
```

Since the `scale` variable is an integer, `%1$d` would be more appropriate than `%1$s`.

Reply:
Right, thanks 😅